src/Components/Facility/AssetCreate.tsx

```diff
@@ -473,434 +474,417 @@ const AssetCreate = (props: AssetProps) => {
+                    </div>
 
-                  {/* Serial Number */}
-                  <div
-                    className="col-span-6 sm:col-span-3"
-                    ref={fieldRef["serial_number"]}
-                  >
-                    <TextFormField
-                      id="serial-number"
-                      name="serial_number"
-                      label="Serial Number"
-                      value={serial_number}
-                      onChange={(e) => setSerialNumber(e.value)}
-                      error={state.errors.serial_number}
-                    />
-                  </div>
+                    {/* Serial Number */}
+                    <div
+                      className="col-span-6 sm:col-span-3"
+                      ref={fieldRef["serial_number"]}
+                    >
+                      <TextFormField
+                        id="serial-number"
+                        name="serial_number"
+                        label="Serial Number"
+                        value={serial_number}
+                        onChange={(e) => setSerialNumber(e.value)}
+                        error={state.errors.serial_number}
+                      />
+                    </div>
 
-                  <div className="mt-6" />
-                  {sectionTitle("Service Details")}
-
-                  {/* Last serviced on */}
-                  <div
-                    className="col-span-6 sm:col-span-3"
-                    ref={fieldRef["last_serviced_on"]}
-                  >
-                    <label htmlFor="last-serviced-on">Last Serviced On</label>
-                    <TextFormField
-                      name="LastServicedOn"
-                      className="mt-2"
-                      value={last_serviced_on}
-                      error={state.errors.last_serviced_on}
-                      onChange={(date) => {
-                        if (
-                          moment(date.value).format("YYYY-MM-DD") >
-                          new Date().toLocaleDateString("en-ca")
-                        ) {
-                          Notification.Error({
-                            msg: "Last Serviced date can't be in future",
-                          });
-                        } else {
-                          setLastServicedOn(
-                            moment(date.value).format("YYYY-MM-DD")
-                          );
-                        }
-                      }}
-                      type="date"
-                      max={moment(new Date()).format("YYYY-MM-DD")}
-                    />
+                    <div className="mt-6" />
+                    {sectionTitle("Service Details")}
+
+                    {/* Last serviced on */}
+                    <div
+                      className="col-span-6 sm:col-span-3"
+                      ref={fieldRef["last_serviced_on"]}
+                    >
+                      <label htmlFor="last-serviced-on">Last Serviced On</label>
```

Comment:
Why is this label here? We can pass it as label prop of the below textformfield ryt?

Reply:
Yes, you are right. I have fixed this. Kindly review the changes.